src/vmm_asn.ml

```diff
@@ -508,7 +538,11 @@ let unikernel_config =
          @ (required ~label:"fail-behaviour" fail_behaviour)
          @ (required ~label:"cpuid" int)
          @ (required ~label:"memory" int)
-         @ (optional ~label:"blocks" (my_explicit 0 (set_of utf8_string)))
+         @ (optional ~label:"blocks"
+              (my_explicit 0 (set_of
```

Comment:
Here we use `set_of` while for bridges we use `sequence_of` for essentially the same structure. Semantically, I think this makes sense. Maybe we should change bridges to use `set_of` as well if we're making changes anyway?

Reply:
well spotted, indeed this is a relict from back when solo5 didn't had --net:<name>=<tap>, and the order was important. I changed it here and in unikernel_info as well, and force-pushed.